datadog-service/0.15.0/artifacthub-pkg.yml

```diff
@@ -3,7 +3,7 @@ name: datadog-service
 displayName: Datadog Service
 createdAt: '2022-07-12T06:21:51Z'
 description: Keptn SLI provider for datadog
-logoURL: https://imgix.datadoghq.com/img/about/presskit/logo-v/dd_vertical_purple.png
+logoURL: https://raw.githubusercontent.com/keptn-contrib/artifacthub/main/datadog-service/images/datadog-logo.jpg
```

Comment:
@tarangverma https://imgix.datadoghq.com/img/about/presskit/logo-v/dd_vertical_purple.png gives us the datadog logo too. I don't think logo link is the problem (having the logo in our repo is a good idea though). The problem is artifacthub for some reason is not picking our logo or the new service name (`Datadog Service` in place of the old service name `datadog-service`) I think it might be a CI/CD related issue.

Reply:
When I say CI/CD, I mean https://github.com/keptn-contrib/artifacthub/tree/main/.github/workflows